webrtc.html

```diff
@@ -4971,6 +4971,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       encoded and transmitted to a remote peer. When <code>setParameters</code>
       is called on an <code><a>RTCRtpSender</a></code> object, the encoding is
       changed appropriately.</p>
+
+      <p>When sending media, the sender may need to rescale or
+      resample the media to meet various constraints including the
```

Comment:
Maybe s/constraints/requirements/ to avoid confusion with mediacapture constraints?